packages/sanity/src/core/form/inputs/PortableText/object/BlockObject.tsx

```diff
@@ -115,6 +115,7 @@ export function BlockObject(props: BlockObjectProps) {
         onClickingEdit={openItem}
         readOnly={readOnly}
         renderPreview={renderPreview}
+        isOpen={memberItem?.member.open}
```

Comment:
`memberItem?.member.open` must be part of the hook dependency below.

I suggest you do a:

`const isOpen = !!memberItem?.member.open` above this, and then use `isOpen` as hook dependency and variable inside this hook.

Reply:
You're right, weird that my linting didn't complain about the missing dependency 🤔 
I'll update these 👍

Reply:
Yeah, maybe you have to restart the TS server inside VSCode.